lib/maintenance_tasks.rb

```diff
@@ -94,4 +84,29 @@ module MaintenanceTasks
   #
   #  @return [ActiveSupport::Duration] the threshold in seconds after which a task is considered stuck.
   mattr_accessor :stuck_task_duration, default: 5.minutes
+
+  class << self
+    DEPRECATION_MESSAGE = "MaintenanceTasks.error_handler is deprecated and will be removed in the next release. " \
+      "Instead, reports will be sent to the Rails error reporter. Do not set a handler and subscribe" \
+      "to the error reporter instead."
+
+    # no-doc
```

Comment:
Should we use the [`@deprecated` directive](https://rubydoc.info/gems/yard/file/docs/Tags.md#deprecated) for these instead?